README.md

```diff
@@ -178,6 +179,13 @@ The `barColors` option allows you to customize the color of each progress bar st
 | progress | #68a9ef | ![#68a9ef](https://placehold.it/15/68a9ef/000000?text=+) |
 | ignored | #7a7a7a | ![#7a7a7a](https://placehold.it/15/7a7a7a/000000?text=+) |
 
+The `defaultMessages` option allows you to override some default messages in the UI. At the moment these are:
+
+| Message Id | When Shown | Default Value |
+|-------|----------|:-------------:|
+| waiting | Task is waiting to start | 'Waiting for task to start...'
+| error | Task has started but reports no progress | 'Task started...'
```

Comment:
I believe the `error` message ID should be switched to `started`, as that's what you've used in the code.

Reply:
Oops, good catch! Copy/paste fail. https://github.com/czue/celery-progress/pull/89/commits/213a634594526f9efb5c2fd90a07d9f9d6de0768